### PR TITLE
added VA108xx support

### PR DESCRIPTION
**Pull request by robamu**
As suggested in https://github.com/rust-embedded/awesome-embedded-rust/pull/351 .
I can also add the new device driver crate (max116xx 10-bit) but I think this is already covered by https://github.com/rust-embedded/awesome-embedded-rust/pull/351

### CODE REVIEW

File: content/newsletter-next.md [hunk 1]
```diff
@@ -45,7 +45,11 @@ If you have an embedded project or blog post you would like to have featured in
 - [embedded-hal] 1.0.0-alpha.6 released, with a number of new features and
     changes as we work towards the final 1.0 release. Most notably,
     CAN (Controller Area Network) traits were added.
-
+- New Rust Embedded ecosystem for the Vorago VA108xx family of devices:
```
Comment (eldruin) [inline content/newsletter-next.md:48]:
```suggestion
- New Rust Embedded ecosystem for the radiation-hardened Vorago VA108xx family of devices:
```
space sounds cool, hence this suggestion

**Review by eldruin — COMMENTED** [review]
Great, thank you!
I think it would be worth adding an entry about the ADC driver as well. This has more visibility than one entry in the a-e-r list.

**Comment by robamu** [comment]
Okay, I added it as well

**Review by eldruin — APPROVED** [review]
Great, thank you!
bors r+

**Pull request merged**